test_scripts/API/VehicleData/GetVehicleData/002_RPC_parameter_DISALLOWED_by_policies_flow.lua

```diff
@@ -0,0 +1,74 @@
+---------------------------------------------------------------------------------------------------
+-- Item: Use Case: request is allowed but parameter of this request is NOT allowed by Policies
+--
+-- Requirement summary:
+-- [GetVehicleData] As a mobile app wants to send a request to get the details of the vehicle data
+--
+-- Description:
+-- In case:
+-- 1) mobile application sends valid GetVehicleData to SDL and this request is allowed by Policies but RPC parameter is not allowed
+-- SDL must:
+-- 1) SDL responds DISALLOWED, success:false to mobile application and doesn't transfer this request to HMI
+
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/API/VehicleData/commonVehicleData')
+local commonTestCases = require("user_modules/shared_testcases/commonTestCases")
+
+--[[ Local Variables ]]
+
+local rpc = {
+  name = "GetVehicleData",
+  params = {
+    engineOilLife = true
+  }
+}
+
+-- Function which removes engineOilLife parameter from specified func group and rpc
+local function ptu_update_func(tbl)
+  local params = tbl.policy_table.functional_groupings["Emergency-1"].rpcs["GetVehicleData"].parameters
+  for index, value in pairs(params) do
+    if ("engineOilLife" == value) then params[index] = nil end
+  end
+end
+
+local function delayExp()
```

Comment:
@AByzhynar As I could see this function is not used more so you can remove it and remove commented call

Reply:
@AKalinich-Luxoft Thanks . Will delete it.

Reply:
@AKalinich-Luxoft Done in 891857a555e71a421a01111967d75e5bf7fbc92b